ast/src/main/java/com/graphicsfuzz/common/glslversion/ShadingLanguageVersion.java

```diff
@@ -177,6 +177,16 @@ static ShadingLanguageVersion webGlFromVersionString(String versionString) {
 
   boolean supportedIntBitsToFloat();
 
+  /**
+   * Integer Functions are a set of built-in functions (think sqrt, max, min, etc.) that
```

Comment:
I think your examples are wrong.

In the GLSL ES 3.20 specification, the integer functions are things like uaddCarry.  Can you check and re-work?  sqrt, max, min, etc., work on various types - not just integers, and they are not in the "Integer Functions" section of the spec.

Reply:
Sorry - the way I meant that was as examples of built-in functions in general, not as examples of integer functions specifically. The purpose was to explain what a built-in would be considered as to someone coming from a different language (Java, C++, etc.), but I suppose it's not really needed in this context.